examples/pipelines/simple_pipeline/simple_pipeline_fondant.py

```diff
@@ -0,0 +1,42 @@
+"""Pipeline used to create a stable diffusion dataset from a set of given images."""
+import logging
+
+from config.general_config import KubeflowConfig
+from config.components_config import (
+    LoadFromHubConfig,
+    ImageFilterConfig,
+)
+
+from fondant.pipeline import FondantComponentOp, FondantPipeline, FondantClient
```

Comment:
Everything is called Fondant, might be a bit much

Reply:
Yes, would drop it everywhere (and maybe only keep an F). But we can do that in a separate PR.

Reply:
Makes sense indeed. Created a [ticket](https://github.com/orgs/ml6team/projects/1/views/1?pane=issue&itemId=27797215) for it

Reply:
Why not just call them `ComponentOp`, `Pipeline`, etc?

Reply:
I wanted to make it a bit more explicit to avoid confusion with KFP native objects since the Fondant pipeline is essentially a wrapper around it